Hi3Helper.Core/Lang/en.json

```diff
@@ -243,8 +243,8 @@
     "OverlayNotInstalledSubtitle": "Please download/update the game first on the game's launcher page!",
     "OverlayGameRunningTitle": "Game is Currently Running!",
     "OverlayGameRunningSubtitle": "Please close the game first to use this feature!",
-    "OverlayFirstTimeTitle": "You haven't play this game for the first time!",
-    "OverlayFirstTimeSubtitle": "Please run the game first and then come back to use this feature.",
+    "OverlayFirstTimeTitle": "You haven't launched the game yet!",
+    "OverlayFirstTimeSubtitle": "Please run the game first to generate the necessary files, then come back here to use this feature.",
```

Comment:
Actually, the launcher is looking for a registry and not a file. To match the context, can we change it to:
``Please run the game first to generate the necessary files, then come back here to use this feature.``

And for ``OverlayFirstTimeTitle``:
``You have never played this game before!``

Reply:
Sounds good, I'll make the changes once I'm home. In the meanwhile, I've been trying to use that settings menu but it is never available... Could it be that the registry key location has changed?

Reply:
The game settings menu is only availabe for Honkai Impact 3rd. Genshin doesn't have that.

Reply:
> The game settings menu is only available for Honkai Impact 3rd. Genshin doesn't have that.

Correct, but it doesn't display the settings even after launching HI3 once.

I'll double-check and let you know once I get home.

Reply:
Ok, fixed the strings, but had a chance to take a look at the registry issues. It seems that the keys you need are there, but they are never loaded in your function? `InitRegKey()`'s `RegKey` value is `null`.

Tried logging it to see what was up:
![image](https://user-images.githubusercontent.com/9833218/206794049-b8e4d103-8c1f-4c2a-97de-951cf0441ed2.png)

Here's what my registry contains: 
[hi3_keys.zip](https://github.com/neon-nyan/Collapse/files/10198100/hi3_keys.zip)

Let me know if you want to open an issue on this or if it's just a corrupted installation.

Reply:
Sorry for my very late reply on this since I'm just back.
This behavior is actually expected. The launcher doesn't detect the registry value since the registry would only be exist if you have once entered the game and play some inside the game (not only logging into the game and then quit).

Reply:
Other than that, I think we are pretty much complete with this PR and I'm going to merge this for now.
Thank you for your contribution! 😃